near-sdk/src/environment/env.rs

```diff
@@ -948,6 +1721,19 @@ pub fn storage_read(key: &[u8]) -> Option<Vec<u8>> {
 }
 /// Removes the value stored under the given key.
 /// If key-value existed returns `true`, otherwise `false`.
+///
```

Comment:
should mention usage in 
https://github.com/near/near-sdk-rs/tree/master/near-contract-standards/src/upgrade/mod.rs